test/util/cluster/cluster.go

```diff
@@ -225,3 +231,11 @@ func IsClusterStable(clients *testclient.ClientSet) (bool, error) {
 
 	return true, nil
 }
+
+func IsEofError(err error) bool {
```

Comment:
Why the function and not comparing the error to the value?
I'd say it's more clear and the function does not add information.

Reply:
Done